docs/intro-react-native-components.md

```diff
@@ -0,0 +1,63 @@
+---
+id: intro-react-native-components
+title: Core Components and Native Components
+---
+
+React Native is an open source framework for building Android and iOS applications using [React](https://reactjs.org/) and the app platform’s native capabilities. With React Native, you use JavaScript to access your platform’s APIs as well as to describe the appearance and behaviour of your UI using React components, bundles of reusable, nestable code. You can learn more about React in the next section. But first, let's cover how components work in React Native.
+
+## Views and mobile development
+
+In Android and iOS development, a **view** is the basic building block of UI: a small rectangular element on the screen which can be used to display text, images, or respond to user input. Even the smallest visual elements of an app, like a line of text or a button, are kinds of views. Some kinds of views can nest other views. It’s views all the way down!
+
+## Native Components
+
+In Android development, you write views in Java or Kotlin; in iOS development, you use Objective-C or Swift. With React Native, you can invoke these views with JavaScript using React components. At runtime, React Native creates the corresponding Android and iOS views for those components. Because React Native components are backed by the same views as Android and iOS, React Native apps look, feel, and perform like any other apps. We call these platform-backed components Native Components.
+
+## Core Components
+
+React Native comes with a suite of built-in components ready for you to use. These Core Components include:
```

Comment:
After reading this, I don't have a super clear idea of what's the difference between Native and Core components, whether it matters, or whether they're the same thing. Maybe just clarifying that with a few words before jumping into the table could help.

Reply:
So I moved things around a bit so that the section was a little tighter and introduced the idea of "hey, you can make your own Native Components—but to get started, we're gonna work with this of premade Native Components that come with RN—the Core Components!"